aip/general/0122.md

```diff
@@ -43,6 +43,14 @@ the leading slash:
     characters that require URL-escaping, or characters outside of ASCII.
   - If Unicode characters can not be avoided, resource names **must** be stored
     in Normalization Form C (see [AIP-210][]).
+- Resources **must** expose a `name` field that contains its resource name.
+  - Resources **may** provide the resource ID as a separate field (e.g.
```

Comment:
Should we make the field suffix of `_id` a "must"?

Reply:
Right now it has nothing. Thinking "round toward should". Done.